src/map/atcommand.cpp

```diff
@@ -7828,24 +7829,26 @@ ACMD_FUNC(whodrops)
 
 ACMD_FUNC(whereis)
 {
-	struct mob_db *mob_array[MAX_SEARCH];
-	int count;
-	int i, j, k;
+	MobID mob_ids[MAX_SEARCH] = {0};
+	int count = 0;
+	const int MAX_SPAWN_DISP = 10;
```

Comment:
Should we remove that? It's currently not used and just displays all spawns.

Reply:
Yeah I think the MAX_SEARCH value is what the true cap is.